Source/SwiftLintBuiltInRules/Rules/Metrics/NestingRule.swift

```diff
@@ -18,23 +18,18 @@ struct NestingRule: Rule {
 private struct ValidationData {
     private(set) var typeLevel: Int = -1
     private(set) var functionLevel: Int = -1
-    private var declStack = Stack<any DeclSyntaxProtocol>()
+    private(set) var functionOrNotStack = Stack<Bool>()
```

Comment:
I think, they can all be `private` without `(set)`.

Reply:
I made all properties private scoped! Could you review it?